app/views/scenarios/_form.html.erb

```diff
@@ -50,6 +50,18 @@
     <div class="col-md-4">
       <div class="form-group">
         <div>
+          <%= f.label :icon %>
+          <%= f.select(:icon, options_for_select(YAML.load_file(File.expand_path('config/icons.yml'))), {},
```

Comment:
I think we should cache the icons array somewhere, so that the YAML file does not need to be loaded every time the form is rendered. `Scenario.icons` would be an option.